changelogs/270-flatpak-argument-types.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- flatpak - add argument types to module documentation.
```

Comment:
You can remove the changelog here; we generally don't add changelog fragments for docs-only changes.

Reply:
Thanks. I've removed the changelog in the latest revision.